stl/inc/ostream

```diff
@@ -454,6 +454,13 @@ public:
         return *this;
     }
 
+#if _HAS_CXX23 // P1147R1 "Printing volatile Pointers"

```

Comment:
We don't typically document the vehicle by which functionality was added to the Standard:
```suggestion
#if _HAS_CXX23
```